docs/README.md

```diff
@@ -16,6 +16,8 @@ At the beginning of every session a new George Backend team member will join whi
 In the Remote-Pair-Programming Sessions you need to share your screen so that we can follow your progress.
 In case you have a dual monitor setup, please make sure that you also share your browser window.
 Of course, you can use the web to research in case you need to look up something.
+You are also allowed to use AI powered tools (e.g. ChatGPT, IntelliJ AI Assistant) as an extension to refine your *own* 
```

Comment:
Maybe mention also that googling is allowed, but not for the explicit solution of wordcount.
Same applies also to ai tools.

Reply:
> Of course, you can use the web to research in case you need to look up something.

Googling allowed is already mentioned 😄 

> but not for the explicit solution of wordcount

👍

Reply:
[ed8f6e3](https://github.com/george-labs/george-backend-hiring/pull/356/commits/ed8f6e38c49f3344031df7a97771e9fed441c455)